\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{1}{|r}|}
\hline
Assembly & SRRoutput \\ \hline
\# contigs ($\geq$ 0 bp) & 22 \\ \hline
\# contigs ($\geq$ 1000 bp) & 22 \\ \hline
\# contigs ($\geq$ 5000 bp) & 22 \\ \hline
\# contigs ($\geq$ 10000 bp) & 22 \\ \hline
\# contigs ($\geq$ 25000 bp) & 18 \\ \hline
\# contigs ($\geq$ 50000 bp) & 17 \\ \hline
Total length ($\geq$ 0 bp) & 12051270 \\ \hline
Total length ($\geq$ 1000 bp) & 12051270 \\ \hline
Total length ($\geq$ 5000 bp) & 12051270 \\ \hline
Total length ($\geq$ 10000 bp) & 12051270 \\ \hline
Total length ($\geq$ 25000 bp) & 11978000 \\ \hline
Total length ($\geq$ 50000 bp) & 11945303 \\ \hline
\# contigs & 22 \\ \hline
Largest contig & 1495363 \\ \hline
Total length & 12051270 \\ \hline
Reference length & 12157105 \\ \hline
Estimated reference length & - \\ \hline
GC (\%) & 38.32 \\ \hline
Reference GC (\%) & 38.15 \\ \hline
N50 & 802231 \\ \hline
NG50 & 802231 \\ \hline
N90 & 348143 \\ \hline
NG90 & 348143 \\ \hline
auN & 852710.3 \\ \hline
auNG & 845286.9 \\ \hline
L50 & 6 \\ \hline
LG50 & 6 \\ \hline
L90 & 14 \\ \hline
LG90 & 14 \\ \hline
\# total reads & - \\ \hline
\# left & - \\ \hline
\# right & - \\ \hline
Mapped (\%) & - \\ \hline
Reference mapped (\%) & - \\ \hline
Properly paired (\%) & - \\ \hline
Reference properly paired (\%) & - \\ \hline
Avg. coverage depth & - \\ \hline
Reference avg. coverage depth & - \\ \hline
Coverage $\geq$ 1x (\%) & - \\ \hline
Reference coverage $\geq$ 1x (\%) & - \\ \hline
\# large blocks misassemblies & - \\ \hline
\# misassemblies & 86 \\ \hline
\# misassembled contigs & 18 \\ \hline
Misassembled contigs length & 11372283 \\ \hline
\# local misassemblies & 31 \\ \hline
\# scaffold gap ext. mis. & 0 \\ \hline
\# scaffold gap loc. mis. & 0 \\ \hline
\# structural variations & - \\ \hline
\# possible TEs & - \\ \hline
\# unaligned mis. contigs & 0 \\ \hline
\# unaligned contigs & 0 + 14 part \\ \hline
Unaligned length & 70803 \\ \hline
Genome fraction (\%) & 96.901 \\ \hline
Duplication ratio & 1.015 \\ \hline
Avg contig read support & - \\ \hline
\# N's per 100 kbp & 0.00 \\ \hline
\# mismatches per 100 kbp & 211.81 \\ \hline
\# indels per 100 kbp & 21.60 \\ \hline
\# genomic features & - \\ \hline
\# operons & - \\ \hline
Complete BUSCO (\%) & - \\ \hline
Partial BUSCO (\%) & - \\ \hline
\# predicted genes (unique) & - \\ \hline
\# predicted genes ($\geq$ 0 bp) & - \\ \hline
\# predicted genes ($\geq$ 300 bp) & - \\ \hline
\# predicted genes ($\geq$ 1500 bp) & - \\ \hline
\# predicted genes ($\geq$ 3000 bp) & - \\ \hline
\# predicted rRNA genes & - \\ \hline
Largest alignment & 765542 \\ \hline
Total aligned length & 11956409 \\ \hline
NA50 & 267325 \\ \hline
NGA50 & 267325 \\ \hline
NA90 & 107862 \\ \hline
NGA90 & 95271 \\ \hline
auNA & 349090.8 \\ \hline
auNGA & 346051.8 \\ \hline
LA50 & 13 \\ \hline
LGA50 & 13 \\ \hline
LA90 & 41 \\ \hline
LGA90 & 42 \\ \hline
K-mer-based compl. (\%) & - \\ \hline
K-mer-based cor. length (\%) & - \\ \hline
K-mer-based mis. length (\%) & - \\ \hline
\# k-mer-based misjoins & - \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
